MicroProfile-Rest-Client/pom.xml

```diff
@@ -54,8 +54,8 @@
     <name>MicroProfile Rest Client TCK Runner - Parent</name>
 
     <properties>
-        <microprofile.rest.client.tck.version>2.0</microprofile.rest.client.tck.version>
-        <microprofile.rest.client.api.version>2.0</microprofile.rest.client.api.version>
+        <microprofile.rest.client.tck.version>3.0</microprofile.rest.client.tck.version>
```

Comment:
Versions are already upgraded in `microprofile-5.0` branch

Reply:
I already merged changes from the microprofile-5.0